src/coreclr/pal/src/misc/fmtmessage.cpp

```diff
@@ -73,7 +73,18 @@ static LPWSTR FMTMSG_GetMessageString( DWORD dwErrCode )
         }
         else
         {
-            swprintf_s(lpRetVal, MAX_ERROR_STRING_LENGTH, W("Error %u"), dwErrCode);
+            WCHAR msg[] = W("Error ");
```

Comment:
A nit - since we call sprintf_s anyways, it seems we could make this stuff simpler by printing the `Error ` part using the sprintf_s too:
```c++
            char errorString[sizeof("Error 4294967295")];
            int cnt = sprintf_s(errorString, sizeof(errorString), "Error %u", dwErrCode);

            cnt++; // +1 for null terminator

            // Widening characters is okay here because they are the
            // same in both char and WCHAR.
            for (int i = 0; i < cnt; ++i)
                lpRetVal[i] = (WCHAR)errorMessage[i];
```

Reply:
Ugh... I missed the forest through the trees. Yes, I should just do that.